Working branch

skjohansen opened the pull request.

AnorZaken committed:
mer delete

nordeke committed:
Create UserStories.md

nordeke committed:
Update UserStories.md

nordeke committed:
Update UserStories.md

nordeke committed:
Update UserStories.md

nordeke committed:
Update UserStories.md

AnorZaken committed:
Update UserStories.md

AnorZaken committed:
Update UserStories.md

AnorZaken committed:
Update UserStories.md

nordeke committed:
Update UserStories.md

AnorZaken committed:
Update UserStories.md

nordeke committed:
Update Program.cs

nordeke committed:
Update Program.cs

Giss47 committed:
Update UserStories.md

User stories structuring

Giss47 committed:
Update UserStories.md

/Efter Jag läst mer om UserStories, verkar att de skulle ser ut så här.

Giss47 committed:
Update UserStories.md

Behöver inte vara på samma rad!!

Giss47 committed:
Created Palyer Class

Giss47 committed:
white space

nordeke committed:
Session2

nordeke committed:
Die gone, och lite annat

nordeke committed:
gg

AnorZaken committed:
GG

AnorZaken committed:
Merge branch 'WorkingBranch' of https://github.com/PGBFDH18/ludo-console-game-madmax into WorkingBranch

AnorZaken committed:
Made a new Engine project (MadEngine)

Wrote a shitton of code and tested none of it - what could go wrong?
Anyway in theory the engine is done. GUI remaining.

AnorZaken committed:
Mer info metoder för UI

Glömde förra gången fixa så att UI kan fråga vad som finns på en position, och kunna fråga var pjäser är som inte tillhör CurrentPlayer kan ju vara bra också...

AnorZaken committed:
Added BoardInfo.Size and more documentation.

AnorZaken committed:
Added the ability to load / save.

CarljohanDanling reviewed (commented):
Nice jobb! 👍 

Jag kommer dock inte så långt i spelsekvensen. Om jag, som spelare1, slår t.ex. en 5:a så slås en 5:a hela tiden efter det. Vet inte vart i koden det kan behöva ändras.

Ni verkar ha mycket kod som är funktionsduglig men som inte kommer i användning då man aldrig kommer ur boet. Jag skulle gärna vilja spela igen när ni ordnat det 👍 Peace amigos.

skjohansen commented on src/LudoByMadMax.sln:
```diff
@@ -3,9 +3,11 @@ Microsoft Visual Studio Solution File, Format Version 12.00
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "LudoByMadMax", "LudoByMadMax\LudoByMadMax.csproj", "{F5BA6B91-14FC-4CDD-B8CE-97ABA1AAFD27}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Engine", "Engine\Engine.csproj", "{54AA2D91-9443-4864-95E5-7D524313ED40}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Engine", "Engine\Engine.csproj", "{54AA2D91-9443-4864-95E5-7D524313ED40}"
```
Can this be deleted? It seems that MadEngine is the one which is used

skjohansen commented on src/Engine2/Session.cs:
```diff
@@ -0,0 +1,337 @@
+namespace MadEngine
+{
+    // var försiktiga så ni inte blandar ihop distance och position!
+    internal class Session : ISession
```
👍

skjohansen reviewed (commented):
Looks good
For some reason do I always roll 3, I have a theory that you are not generating a new random number, but did not find the root of course..
Remove the unused project :)

skjohansen commented:
> 
> 
> Nice jobb! 👍
> 
> Jag kommer dock inte så långt i spelsekvensen. Om jag, som spelare1, slår t.ex. en 5:a så slås en 5:a hela tiden efter det. Vet inte vart i koden det kan behöva ändras.
> 
> Ni verkar ha mycket kod som är funktionsduglig men som inte kommer i användning då man aldrig kommer ur boet. Jag skulle gärna vilja spela igen när ni ordnat det 👍 Peace amigos.

Can you approve the pull request?

AnorZaken committed:
Made the game playable!
- Added the missing code in the console project. (which now uses C#7.2)
- Added a few new useful features to the game API. (e.g. a starting-player argument)
- Fixed various engine bugs.
- Removed the old unused Engine project.

AnorZaken commented:
Fixed all the issues. Game is now playable.

(Hint: Just pressing Enter when asked for a name uses the default name of Player#, so you don't have to type so much when testing.)

AnorZaken committed:
Very minor changes

AnorZaken committed:
Can now play with bots
